tools/online_editor/src/index.ts

```diff
@@ -120,7 +171,92 @@ function create_share_menu(editor: EditorWidget): Menu {
             navigator.clipboard.writeText(share_url);
         },
     });
+    commands.addCommand("slint:create_gist", {
+        label: "Export to github Gist",
+        iconClass: "fa-brands fa-github",
+        mnemonic: 1,
+        isEnabled: () => {
+            return localStorage.getItem(local_storage_key_github_token) != null;
+        },
+        execute: async () => {
```

Comment:
nitpick style: it might be cleaner to move the gist specific saving and loading to a different file.